src/ds/files.h

```diff
@@ -89,6 +97,92 @@ namespace files
     return nlohmann::json::parse(v.begin(), v.end());
   }
 
+  /**
+   * @brief Tries to glob files and parse them as x509 certificates
+   *
+   * @param file the path to glob
+   * @param optional determines behaviour in the case where the file does not
+   * exist. If true, an empty JSON object is returned. If false, the process
```

Comment:
This comment isn't accurate. This function potentially returns an empty vector, not an empty json object. Also since we're talking about globs, "in the case where the file does not exist" doesn't really make sense (I think its "in the case that no file matches the glob"?)

Reply:
Good spot!